From bbdd65e86fd6c81e633edd38d2dfb626b221dbef Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Mon, 13 May 2024 18:00:33 +0800
Subject: [PATCH 024/109] WIN2030-15099:fix(hf_common.c):_write was disabled
 unexpectly

Changelogs:
1._write(int fd, char *ch, int len) is used for printf redirection.
  It must be enabled.

Change-Id: I71a8c86bfc5e14405a89b3da7be9c9a743ed6b06
---
 Core/Src/hf_common.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Core/Src/hf_common.c b/Core/Src/hf_common.c
index 080600f..a9320cf 100644
--- a/Core/Src/hf_common.c
+++ b/Core/Src/hf_common.c
@@ -138,7 +138,7 @@ void hexstr2mac(uint8_t *mac, char *hexstr)
 		hexstr += 2;
 	}
 }
-#if 0
+
 int _write(int fd, char *ch, int len)
 {
 	uint8_t val = '\r';
@@ -151,7 +151,6 @@ int _write(int fd, char *ch, int len)
 	}
 	return length;
 }
-#endif
 
 
 void es_eeprom_wp(uint8_t flag)
-- 
2.25.1

